en-US/Grammar.xml

```diff
@@ -471,10 +471,15 @@
 				</thead>
 				 <tbody>
 					<row>
-						<entry> The individual member of the social community often receives information via visual, symbolic channels. </entry>
-						 <entry> People read. (Translation by Richard Feynman.) </entry>
+						<entry> The <command>fsck</command> utility performs a file system verification on the XFS file system residing on the <filename>/dev/vdb1</filename> partition. </entry>
+						 <entry> The <command>fsck</command> utility verifies the XFS file system on the <filename>/dev/vdb1</filename> partition. </entry>
 
 					</row>
+					<row>
+						<entry> It is possible that the scheduled snapshot takes some time to be created. </entry>
+						 <entry> Creating the scheduled snapshot might take time. </entry>
```

Comment:
Everything takes time. Can we elaborate a bit? Something like "Creating the snapshot will take more or less time depending on the size of the system." Or maybe something different altogether?

Reply:
Replaced with a different example.